lib/config/presets/internal/default.ts

```diff
@@ -421,7 +421,10 @@ export const presets: Record<string, Preset> = {
     ],
   },
   pinDigestsDisabled: {
-    description: 'Disable pinning of Docker dependency digests.',
+    description: `Disable pinning of Docker dependency digests.
+
+Note: This preset does not revert the digest pinning of the \`docker:pinDigests\` preset.
```

Comment:
```suggestion
Note: This preset does not revert the digest pinning set from within the `packageRules`, when set at root level or via extends. 
```

Reply:
With the suggested version, I see the risk, that less knowledgeable users still try to use this preset to "revert" the `docker:pinDigests` from e.g. `config:best-practices` and do not fully understand the implications of the sentence you suggest.
Shall I apply your suggestion anyway?